### ISSUE TITLE
FileUpload: Missing form label (a11y)

### ISSUE THREAD

**Issue by SoyDiego** [primefaces/primeng#15137]
### Describe the bug

The component FileUpload is missing the labels.

You can check it using Wave extension:

![image](https://github.com/primefaces/primeng/assets/19764334/2027ee4a-bebc-47fb-8f06-298061adbe8b)

### Environment

-

### Reproducer

https://primeng.org/fileupload/

### Angular version

Last version

### PrimeNG version

Last version

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

20

### Steps to reproduce the behavior

1. Go to https://primeng.org/fileupload/
2. Open Wave extension for accessibility.
3. Will see the problems with the hidden inputs

### Expected behavior

Solve the problem with Wave extension
